widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.kt

```diff
@@ -1553,7 +1560,11 @@ internal class ChatController(
 
     override fun newEngagementLoaded(engagement: OmnicoreEngagement) {
         Logger.d(TAG, "newEngagementLoaded")
-        subscribeToMessages()
+
+        // Loading chat history again on engagement start in case it was an-authenticated visitor that restored ongoing engagement
```

Comment:
`an-authenticated` => `un-authenticated` or `de-authenticated`
But it seems to me that you mean `re-authenticated`:
`... in case it the visitor was re-authenticated and ongoing engagement is being restored.`

Besides, I would add step by step description of this case because it's still not clear why we need something special in addition to standard load chat history flow.

Reply:
New comments were added.